model/clusters_mgmt/v1/cluster_registry_config_type.model

```diff
@@ -0,0 +1,88 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// ClusterRegistryConfig describes the configuration of registries for the cluster.
+// Its format reflects the OpenShift Image Configuration, for which docs are available on
+// [docs.openshift.com](https://docs.openshift.com/container-platform/4.16/openshift_images/image-configuration.html)
+// ```json
+// {
+//    "registry_config": {
+//      "registry_sources": {
+//        "blocked_registries": [
+//          "badregistry.io",
+//          "badregistry8.io"
+//        ]
+//      }
+//    }
+// }
+// ```
+//
+struct ClusterRegistryConfig {
+	// PlatformAllowlist contains a reference to a RegistryAllowlist which is a list of internal registries
+	// which needs to be whitelisted for the platform to work. It can be omitted at creation and 
+	// updating and its lifecycle can be managed separately if needed.
+	PlatformAllowlist RegistryAllowlist
+	// A map containing the registry hostname as the key, and the PEM-encoded certificate as the value,
+	// for each additional registry CA to trust.
+	AdditionalTrustedCa [String]String
```

Comment:
```suggestion
	AdditionalTrustedCA [String]String
```

minor nit. Upper case `A` to match Openshift doc.
Same considerations for the other instances.